backend/src/main/java/com/huellapositiva/infrastructure/orm/repository/JpaEmailConfirmationRepository.java

```diff
@@ -21,6 +22,6 @@
 
     @Modifying
     @Transactional
-    @Query("UPDATE EmailConfirmation ec SET ec.hash = :hash WHERE ec.email = :email")
-    Integer updateHashByEmail(@Param("email") String email, @Param("hash") String hash);
+    @Query("UPDATE EmailConfirmation ec SET ec.hash = :hash, ec.createdOn = :createdOn WHERE ec.email = :email")
```

Comment:
Hibernate should take care of updating the updatedOn column every time you make changes to the entity